src/components/form/is-favorite-pill.spec.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+
```

Comment:
Also, I would like to discuss with you usage of empty lines thing :) I don't mind to adopt any code styling rules, just would like to have those rules and write similarly.